contents/customers/rayfit.md

```diff
@@ -0,0 +1,60 @@
+
+That solved the infrastructure problem. Next was ensuring the entire team was on board.
+
+## Getting over the learning curve with PostHog AI
+For Rayfit, the early friction wasn’t collecting data, it was making sure everyone could access it. "PostHog can present as really technical," Alan explains. “Our CEO and designer had used Mixpanel and Amplitude before — which felt more approachable since PostHog is built for engineers.”
+
+Their data model added another layer. Not everyone on the team is fluent in SQL, but their workflow depends on it. “A lot of the data we use for the generated workouts lives in the app itself,” Alan says. “It wouldn't make sense to send a unique event for each one. With the warehouse we can join it to a person and learn more about their behaviors.”
+
+That’s where [PostHog AI](/docs/posthog-ai) comes in. For Alan, SQL is second nature — AI just removes some friction. For the rest of the team, having AI draft and edit queries makes the whole workflow far more accessible. “PostHog AI has actually been a deciding factor in our decision to stick with PostHog,” he told us. “Without it, we probably would have moved to a different analytics package because the learning curve felt so steep.”
+
+<OSQuote
+  customer="rayfit"
+  author="alan_yang"
+  quote={0}
+/>
+
+## Realistic expectations, real results
+As an early adopter of AI products (in and outside of PostHog) Alan is pragmatic about AI capabilities. “All LLMs right now are about 30–50%,” he says. “But it’s a nice problem space because that’s still a win when it gets it right.”
+
+So when PostHog AI doesn’t nail the syntax on the first try, he doesn’t see it as a failure — just part of the workflow. In fact, it mirrors how both Rayfit and PostHog operate. “We're shipping things before they're fully baked just like PostHog,” he says. “So when AI features don't work perfectly, everyone is willing to try again.”
+
+Even 30–50% accuracy is a win when the alternative is teammates getting stuck or pinging Alan for every question. “It’s saved me countless hours because the data we have is so much more self-serve,” he says. “Anyone on the team can ask questions to PostHog AI.”
+
+What surprised him most is how quickly others leaned into it. “Some of the engineers are using PostHog AI way more than me,” Alan admits. “They're building insights and dashboards, and each of them has workflows I never really thought about.”
+
+
+![rayfit moving average chart](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/rayfit_rep_counting_chart_5a99280ef4.png)
+<Caption>Rep Counting Availability (how many exercises in a workout have rep counting available)</caption>
+_PostHog AI helped me add a moving average to this insight. This was really cool to build because it quantifies that we're getting better over time._ 
```

Comment:
I assume this is what this feature does? I don't think we need to repeat the name as it is already in the image + formating
```suggestion

![rayfit moving average chart](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/rayfit_rep_counting_chart_5a99280ef4.png)
<Caption>How many exercises in a workout have their AI rep counting feature available</caption>

_PostHog AI helped me add a moving average to this insight. This was really cool to build because it quantifies that we're getting better over time._ 
```